src/asset/images.js

```diff
@@ -0,0 +1,5 @@
+const images ={
+    logo: require('./logo.1e58b23f.png')
```

Comment:
change the location of the image file and its name like: 
asset/images/logo.png